server/service/store/handler_store_data.go

```diff
@@ -29,8 +29,8 @@ func HandleStoreData(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	cwd, _ := os.Getwd()
-	path := filepath.Join(cwd, dealIDStr)
+	homeDir, _ := os.UserHomeDir()
+	path := filepath.Join(homeDir, dealIDStr)
```

Comment:
Would be better to define `--data-dir` flag, so that users can specify where to store files explicitly.

Reply:
Fixed it in [226cd4a](https://github.com/medibloc/panacea-dep-consumer/pull/2/commits/226cd4a64efd3ddd0a82930e480db34f71ad840e).